backend/controllers/todoControllers.js

```diff
@@ -0,0 +1,72 @@
+import { getDataBase } from '../db/index.js'
+import { validateValue } from '../helpers.js'
+
+const db = getDataBase()
+
+export const getAllTodos = async (req, res) => {
+  try {
+    const allTodos = await db.findAsync({})
+    res.send(allTodos)
+  } catch (error) {
+    res.status(500).send({error: "Failed to receive tasks"})
+  }
+  
+}
+
+export const addTask = async (req, res) => {
+  const body = req.body
+  
+  try {
+    await validateValue(body)
+    const task = await db.insertAsync({
+      text: body.text,
+      is_checked: body.is_checked,
+    })
```

Comment:
This function returns a validated object, so you can use it and pass it directly to `insert`